### PR TITLE
Control settings bug

### PR DESCRIPTION
**Pull request by MatthewHilleryJCU**

### CODE REVIEW

File: ProjectSquishy/src/main/java/ProjectSquishy/utils/DataMapper.java [hunk 1]
```diff
@@ -85,22 +85,28 @@ private void mapControlData(Document controlDoc, Map<String, Player> players) {
             }
 
 
-            settings.setPowerSlide(values.get(0));
-            settings.setAirRoll(values.get(1));
+            if (values.size() != 0 ) {
```
Comment (TomStanley95) [inline ProjectSquishy/src/main/java/ProjectSquishy/utils/DataMapper.java]: Is this an error checking block? Would be better to use try and except (catch)

File: ProjectSquishy/src/main/java/ProjectSquishy/utils/DataMapper.java [hunk 2]
```diff
@@ -85,22 +85,28 @@ private void mapControlData(Document controlDoc, Map<String, Player> players) {
             }
 
 
-            settings.setPowerSlide(values.get(0));
-            settings.setAirRoll(values.get(1));
```
Comment (TomStanley95) [inline ProjectSquishy/src/main/java/ProjectSquishy/utils/DataMapper.java]: Is this an error checking block? Would be better to use try and except (catch)

Comment (TomStanley95) [reply ProjectSquishy/src/main/java/ProjectSquishy/utils/DataMapper.java]: @MatthewHilleryJCU

File: ProjectSquishy/src/main/java/ProjectSquishy/utils/ButtonChecker.java [hunk 3]
```diff
@@ -21,6 +22,7 @@
     private static final String PS4R2 = "/commons/images/0/0e/PlayStation_button_R2.svg";
     private static final String PS4L1 = "/commons/images/6/66/PlayStation_button_L1.svg";
     private static final String PS4L2 = "/commons/images/9/97/PlayStation_button_L2.svg";
+    private static final String PS4L3 = "/commons/images/4/48/PlayStation_button_L3.svg";
```
Comment (MatthewHilleryJCU) [inline ProjectSquishy/src/main/java/ProjectSquishy/utils/ButtonChecker.java]: Hello

Comment (MatthewHilleryJCU) [reply ProjectSquishy/src/main/java/ProjectSquishy/utils/ButtonChecker.java]: @TomStanley95

**Pull request merged**